web/less/room.less

```diff
@@ -498,7 +507,26 @@ body {
     padding: 0;
   }
 
-  #addToCall-container,
+    @media @smartphonesLandscape {
+      opacity: 1;
+      bottom: 0;
+      width: (100% + 2px);
+      right: 0;
+      left: 0;
+      background-color: rgba(0, 0, 0, 0.6);
+      background-image: none;
+      padding: 0;
+      width: 70%;
+      padding-left: 15%;
+      padding-right: 15%;
+    }
```

Comment:
I think the media rules don't overlap because the width in the landscape is different from the width in the portrait of devices among android and ios devices.  Notice the max-width of both rules in smartphones and smartphonelandscape

```
@smartphones: ~"only screen and (max-width: 480px), only screen and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5)";
@smartphonesPortrait: ~"@{smartphones} only screen and (orientation:portrait) and (max-width: 480px), only screen and (orientation:portrait) and (-webkit-min-device-pixel-ratio: 1.5, -o-min-device-pixel-ratio: 3/2, min--moz-device-pixel-ratio: 1.5, min-device-pixel-ratio: 1.5)";
@smartphonesLandscape: ~"only screen and (min-device-width : 320px) and (max-device-width : 1024px) and (orientation : landscape) ";


```